src/mitol/payment_gateway/api.py

```diff
@@ -453,6 +480,29 @@ def get_client_configuration():
         }
         return configuration_dictionary
 
+    @staticmethod
+    def get_refund_request(transaction_dict: Dict):
+        """
+        Create a refund request object to from the required attributes in the payment dictionary
+        Args:
+            transaction_dict: (Dict) Data dictionary of the payment response of the Gateway
+            Ideally, The transaction dictionary should have transaction_id, req_amount, req_currency in case of
+            CyberSource.
+        Returns:
+            Object: Refund An object of Refund class that can be used to operate the Refunds
+
+        """
+        # Not doing a .get() on dict because this should break if the required keys are not present in the dictionary
+        transaction_id = transaction_dict["transaction_id"]
+        order_amount = transaction_dict["req_amount"]
+        order_currency = transaction_dict["req_currency"]
```

Comment:
Could you have this raise a custom exception (e.g. something like a `InvalidTransaction`) since `KeyError` can be ambiguous?